src/js/modules/menus.js

```diff
@@ -107,3 +107,22 @@ customElements.define(
   "readthedocs-menu-build-rebuild",
   MenuBuildRebuildElement,
 );
+
+/** Menu item for resyncing all **/
+class MenuSyncAllVersionsElement extends APIEventWrapper {
+  // TODO translate this here or use a different pattern
+  static errorMessage = "There was an error syncing versions";
```

Comment:
I always struggle with this, but I guess it's better to have final dots in sentences.

```suggestion
  static errorMessage = "There was an error syncing versions.";
```